apps/server/src/modules/tool/school-external-tool/controller/api-test/tool-school.api.spec.ts

```diff
@@ -140,7 +140,11 @@ describe('ToolSchoolController (API)', () => {
 				name: externalToolEntity.name,
 				schoolId: postParams.schoolId,
 				toolId: postParams.toolId,
-				status: ToolConfigurationStatusResponse.LATEST,
+				status: toolConfigurationStatusResponseFactory.build({
+					isDisabled: false,
```

Comment:
isDisabled will not be used. In my opinion it would be enough if you set false as default in the factory :)